orb/core/model.py

```diff
@@ -93,34 +158,9 @@ def _allowed(columns=None, context=None):
                     continue
                 else:
                     if hasattr(value, '__json__'):
-                        output[key] = value.__json__()
+                        yield key, value.__json__()
                     else:
-                        output[key] = value
-
-        # don't include the column names
-        if context.returning == 'values':
-            output = tuple(output[column.field()]
-                           for column in context.schemaColumns(self.schema()))
-            if len(output) == 1:
-                output = output[0]
-
-        if context.format == 'text':
-            return projex.rest.jsonify(output)
-        else:
-            return output
-
-    def __iter__(self):
-        """
-        Iterates this object for its values.  This will return the field names from the
-        database rather than the API names.  If you want the API names, you should use
-        the recordValues method.
-
-        :sa         recordValues
-
-        :return     <iter>
-        """
-        for col in self.schema().columns().values():
```

Comment:
moved most of the `__json__` logic to the `__iter__` so that doing `dict(record)` is effectively the same as doing `record.__json__()` except as it comes to the preferences around the JSON serialization